examples/gno.land/p/demo/simpledao/propstore.gno

```diff
@@ -0,0 +1,173 @@
+package simpledao
+
+import (
+	"errors"
+	"std"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/seqid"
+	"gno.land/p/gov/dao"
+)
+
+var (
+	errMissingProposal = errors.New("proposal is missing")
+	errMissingVote     = errors.New("member has not voted")
+)
+
+const (
+	// maxRequestProposals is the maximum number of
+	// paginated proposals that can be requested
+	maxRequestProposals = 10
+
+	// maxRequestMembers is the maximum number of
+	// paginated members that can be requested
+	maxRequestMembers = 50
+)
+
+// proposal is the internal simpledao proposal implementation
+type proposal struct {
+	author      std.Address // initiator of the proposal
+	description string      // description of the proposal
+
+	executor dao.Executor       // executor for the proposal
+	status   dao.ProposalStatus // status of the proposal
+
+	votes *votes // voting mechanism
+}
+
+func (p *proposal) GetAuthor() std.Address {
+	return p.author
+}
+
+func (p *proposal) GetDescription() string {
+	return p.description
+}
+
+func (p *proposal) GetStatus() dao.ProposalStatus {
+	return p.status
+}
+
+func (p *proposal) GetVotes() []dao.Vote {
+	var (
+		voters = p.votes.getVoters()
+		votes  = make([]dao.Vote, 0, voters.Size())
+	)
```

Comment:
```suggestion
	var (
		voters = p.votes.voters
		votes  = make([]dao.Vote, 0, voters.Size())
	)
```

Reply:
Updated as part of [c05289f](https://github.com/gnolang/gno/pull/2581/commits/c05289f18c7cd5c13ea2baf058ae9f3c8eede71d)